olm-catalog/knative-build.v0.3.0.clusterserviceversion.yaml

```diff
@@ -37,6 +37,7 @@ spec:
           - events
           - serviceaccounts
           - configmaps
+          - services
```

Comment:
Shouldn't this be a problem for upstream as well?

Reply:
https://github.com/knative/build/blob/release-0.3/config/200-clusterrole.yaml looks like these resources are dealt with here

Reply:
Well, It's not clear from this change, but I had to add 'services' here to appease OLM. After showering, I realized that upstream doesn't need this service account to be able to create services, but because of our init container hack, that SA definitely does need that capability. Upstream doesn't really address what perms the person has to have to actually apply the knative-build manifest, but OLM definitely does. Am I making any sense?

Reply:
Note on line 7 of the link you shared, there is no "services" listed. But I need it, because the init-container will attempt to apply these: https://github.com/openshift-cloud-functions/knative-operators/blob/f939d61b0409721ff1a9a44940e8e313cc32e301/etc/hacks/knative-build-0.3.0.yaml#L29-L56